ts/utils/payment.ts

```diff
@@ -122,3 +122,10 @@ export const cleanTransactionDescription = (description: string): string => {
     ? descriptionParts[descriptionParts.length - 1].trim()
     : "";
 };
+
+/**
+ * This function control if card brand is MAESTRO.
+ */
+export const isBrandMaestro = (card: CreditCard): boolean => {
+  return card.brand === "MAESTRO";
```

Comment:
if brand is "maestro" it returns false.
An insensitive-case comparison could be better

Reply:
updated within https://github.com/pagopa/io-app/pull/1351/commits/2771838cffb307cfa630a373dd7734dcbe77aa71